package.json

```diff
@@ -41,6 +42,7 @@
     "react-redux": "^7.1.0",
     "react-router": "^5.1.2",
     "react-router-dom": "^5.1.2",
+    "react-select": "3.0.4",
```

Comment:
@djblue Would it be better to add atlas as a dependency and use this dependency transitively?

Reply:
If you are using the component directly, then we should add a dependency on it.